virtualization_test.go

```diff
@@ -172,6 +173,16 @@ func newVirtualizationMachine(
 		t.Fatal("want CanStart is true")
 	}
 
+	// As it is not possible to intentionally cause errors, only logging is possible.
+	disconnected, err := vm.NetworkDeviceAttachmentWasDisconnected()
+	if err == nil {
+		go func() {
+			for disconnected := range disconnected {
+				log.Println(disconnected)
```

Comment:
observed by this line
https://github.com/Code-Hex/vz/pull/169#issuecomment-2454496208